models/confirmation_test.go

```diff
@@ -229,6 +234,47 @@ func TestConfirmationContextCustomUnmarshaler(s *testing.T) {
 	})
 }
 
+func TestConfirmationCalculatesExpiresAt(t *testing.T) {
+	for cType := range Timeouts {
+		invite, err := NewConfirmation(cType, TemplateNamePasswordReset, USERID)
+		if err != nil {
+			t.Fatalf("expected nil, got %+v", err)
+		}
+		if invite.ExpiresAt == nil || invite.ExpiresAt.IsZero() {
+			t.Errorf("expected non-Zero ExiresAt")
+		}
+	}
+}
+
+func TestConfirmationIsExpired(t *testing.T) {
+	for cType := range Timeouts {
+		invite, err := NewConfirmation(cType, TemplateNameCareteamInvite, USERID)
+		if err != nil {
+			t.Fatalf("expected nil error, got %s", err)
+		}
+		if invite.IsExpired() {
+			t.Errorf("expected false, got true")
```

Comment:
[nit] For test failure messages, consider something more descriptive for debugging the test. For example:
```suggestion
			t.Errorf("expected invite to not be expired")
```
This would be similar for other failure cases you have here.